app/Models/MasterAdmin_Dashboard.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Models;
+
+use CodeIgniter\Model;
+
+
+class MasterAdmin_Dashboard extends Model
+{
+
+    public function getStats($data)
+    {
+        $productId_query = "";
+        $AndQueryBuild = " OR product_id =";
+        $userType = $data['user_type'];
+        $userId = $data['masteradmin_id'];
+
+        $product_ids = explode('#', $data['product_id']);
+
+        foreach ($product_ids as $key => $product_id) {
+
+            if($key == 0){
+                $productId_query .= $product_id;
+            }else{
+                $productId_query .=  $AndQueryBuild.$product_id;
+            }
+
+        }
+        
+        $db = db_connect();
+
+        $query1 = $db->query("SELECT COUNT(*) as total_clients FROM user_master WHERE user_type = $userType");
```

Comment:
Count for following missing 

1. Total Content writer count
2. Total no: of active post 
3. Total no: of archived post

please add queries for the above counts